main/src/ca/uwaterloo/flix/language/phase/unification/BoolTable.scala

```diff
@@ -56,12 +54,12 @@ object BoolTable {
   /**
     * The size a formula (but represented as a type) must have before we try to minimize it.
     */
-  private val Threshold: Int = 10
+  val Threshold: Int = 10
```

Comment:
Maybe these should be moved? But I suppose they have dependencies so they cannot?